<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentwindow</class>
 <widget class="QMainWindow" name="studentwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CEG SQUARE - Student Portal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="welcomeLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Welcome, Student! Order food from CEG canteen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="orderTab">
       <attribute name="title">
        <string>Order Food</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="productsLabel">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Available Food Items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="productsTable">
            <property name="columnCount">
             <number>5</number>
            </property>
            <column>
             <property name="text">
              <string>Product</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Shop</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Price (₹)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Quantity</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Add to Cart</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="cartLabel">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Your Cart</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="cartList"/>
          </item>
          <item>
           <widget class="QLabel" name="totalLabel">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: green;</string>
            </property>
            <property name="text">
             <string>Total: ₹0.00</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="cartButtonsLayout">
            <item>
             <widget class="QPushButton" name="placeOrderButton">
              <property name="styleSheet">
               <string notr="true">background-color: #4CAF50; color: white; font-weight: bold; padding: 8px;</string>
              </property>
              <property name="text">
               <string>Place Order</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearCartButton">
              <property name="styleSheet">
               <string notr="true">background-color: #f44336; color: white; padding: 8px;</string>
              </property>
              <property name="text">
               <string>Clear Cart</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="historyTab">
       <attribute name="title">
        <string>Order History</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="historyLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Your Order History</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="historyTable">
          <property name="columnCount">
           <number>5</number>
          </property>
          <column>
           <property name="text">
            <string>Order ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Shop</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Total</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="logoutButton">
      <property name="styleSheet">
       <string notr="true">background-color: #607D8B; color: white; padding: 8px; margin: 10px;</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <!-- ADD STATUS BAR -->
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
